### ISSUE TITLE
Loss function, conf_focal

### ISSUE THREAD

**Issue by Taruio** [YunYang1994/tensorflow-yolov3#197]
In function `loss_layer()`, I find out that you use a coefficient `conf_focal`. 
I would like to know is this a Focal Loss? Why `alpha` in focal loss is always 1?
Thanks

**Comment by RubenS02** [comment]
Good question, moreover shouldn't we use gamma=0 since the author advised not to use focal loss? 
_Things We tried that didn't work:
Focal loss. We tried using focal loss. It dropped our mAP about 2 points. YOLOv3 may already be robust to the problem focal loss is trying to solve because it has separate objectness predictions and conditional class predictions. Thus for most examples there is no loss from the class predictions? Or something? We aren’t totally sure._ (https://arxiv.org/abs/1804.02767)

**Comment by Taruio** [comment]
@RubenS02 
I saw that in YOLOv3 article too. And alpha=1 means it just use a half of focal loss.
I'm traing both gamma=0 and gamma=2 model. If I get something I'll write it here

**Comment by RubenS02** [comment]
@Taruio Thanks I would appreciate that a lot
